CONTRIBUTING.md

```diff
@@ -8,31 +8,41 @@ Thanks for helping making KEDA better!
 You can easily release a new Helm chart version:
 
 1. Update the version of the Helm chart in `Chart.yaml`
-2. Package the Helm chart
+2. Package the Helm chart. For KEDA:
 ```shell
 $ helm package keda
 Successfully packaged chart and saved it to: C:\Code\GitHub\charts\keda-0.1.0.tgz
 ```
```

Comment:
What about this?
```suggestion
2. Package the Helm chart.
  - For KEDA:
  ```shell
  $ helm package keda
  Successfully packaged chart and saved it to: C:\Code\GitHub\charts\keda-0.1.0.tgz
  ```
  - For HTTP Addon:
  ```shell
  $ helm package http-add-on
  Successfully packaged chart and saved it to: /home/ecomaz/src/keda/charts/keda-add-ons-http-0.2.0.tgz
  ```
```